Resolve the ambiguity of the strprintf::fmt call for size_t type definition.

Upstream commit 4139a0ba1ec87e442ef1408823b07fc739742f9d.

Index: include/strprintf.h
--- include/strprintf.h.orig
+++ include/strprintf.h
@@ -55,14 +55,12 @@ std::string fmt(const std::string& format, const uint6
 	return detail::fmt_impl(format, argument, std::forward<Args>(args)...);
 }
 
-#ifdef __APPLE__
 template<typename... Args>
 std::string fmt(const std::string& format, const std::size_t argument,
 	Args&& ... args)
 {
 	return detail::fmt_impl(format, argument, std::forward<Args>(args)...);
 }
-#endif
 
 template<typename... Args>
 std::string fmt(const std::string& format, const void* argument, Args&& ... args)
